enterprise/backend/manage.sh

```
@@ -4,4 +4,6 @@ export DYLD_FALLBACK_LIBRARY_PATH=/opt/homebrew/lib
 DJANGO_DIR=../../backend
 ENTERPRISE_SETTINGS=enterprise_core.settings
 
+export DJANGO_BASE_DIR="$(cd "$(dirname "$0")" && pwd)"
+
 python3 $DJANGO_DIR/manage.py $@ --settings=$ENTERPRISE_SETTINGS
```

Shellcheck (0.11.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)